packages/cli/lib/commands/create.js

```diff
@@ -8,27 +8,116 @@ const { resolve, join } = require('path');
 const { prompt } = require('prompts');
 const isValidName = require('validate-npm-package-name');
 const { info, isDir, hasCommand, error, trim, warn } = require('../util');
-const { addScripts, install, initGit, isMissing } = require('../lib/setup');
+const { addScripts, install, initGit } = require('../lib/setup');
 
 const ORG = 'preactjs-templates';
 const RGX = /\.(woff2?|ttf|eot|jpe?g|ico|png|gif|webp|mp4|mov|ogg|webm)(\?.*)?$/i;
 const isMedia = str => RGX.test(str);
 const capitalize = str => str.charAt(0).toUpperCase() + str.substring(1);
 
+// Formulate Questions if `create` args are missing
+function requestParams(argv) {
+	return [
+		// Required data
+		{
+			type: argv.template ? null : 'select',
+			name: 'template',
+			message: 'Pick a template',
+			choices: [
+				{
+					value: 'preactjs-templates/default',
+					title: 'Default (JavaScript)',
+					description: 'Default template with all features',
+				},
+				{
+					value: 'preactjs-templates/typescript',
+					title: 'Default (TypeScript)',
+					description: 'Default template with all features',
+				},
+				{
+					value: 'preactjs-templates/material',
+					title: 'Material',
+					description: 'Material template using preact-material-components',
+				},
+				{
+					value: 'preactjs-templates/simple',
+					title: 'Simple',
+					description: 'The simplest possible preact setup in a single file',
+				},
+				{
+					value: 'preactjs-templates/widget',
+					title: 'Widget',
+					description:
+						'Template for a widget to be embedded in another website',
+				},
+				{
+					value: 'custom',
+					title: 'Custom',
+					description: 'Use your own template',
+				},
+			],
+			initial: 0,
+		},
+		{
+			type: prev => (prev === 'custom' ? 'text' : null),
+			name: 'template',
+			message: 'Remote template to clone (user/repo#tag)',
+		},
+		{
+			type: argv.dest ? null : 'text',
+			name: 'dest',
+			message: 'Directory to create the app',
+		},
+		// Extra data / flags
+		{
+			type: argv.name ? null : 'text',
+			name: 'name',
+			message: 'The name of your application',
+		},
+		{
+			type: argv.force ? null : 'confirm',
```

Comment:
should this only prompt if the directory exists?

Reply:
That was not in the original code but I can add the check

Reply:
added more cases for the question flow, see the PR description